melodic/distribution.yaml

```diff
@@ -4635,6 +4635,31 @@ repositories:
       url: https://github.com/ros-drivers/rosserial.git
       version: melodic-devel
     status: maintained
+  rotors_simulator:
+    doc:
+      type: git
+      url: https://github.com/ethz-asl/rotors_simulator.git
+      version: 2.2.2
+    release:
+      packages:
+      - rotors_comm
+      - rotors_control
+      - rotors_description
+      - rotors_evaluation
+      - rotors_gazebo
+      - rotors_gazebo_plugins
+      - rotors_hil_interface
+      - rotors_joy_interface
+      - rotors_simulator
+      - rqt_rotors
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/ethz-asl/rotors_simulator-release.git
+      version: 2.2.2-0
+    source:
+      type: git
+      url: https://github.com/ethz-asl/rotors_simulator.git
+      version: 2.2.2
```

Comment:
Oh, right.  The problem is that we don't generally consider tags to be valid for source/doc entries, only branches.  I'll suggest you retarget this to master.

Reply:
I believe that doc is actually ok with a tag but is not recommended, but source does need a branch.

Reply:
fixed - thanks much for your hint!